src/strings/en-us.json

```diff
@@ -93,7 +91,6 @@
     "ButtonHelp": "Help",
     "ButtonHome": "Home",
     "ButtonInfo": "Info",
-    "ButtonLearnMore": "Learn more",
```

Comment:
https://github.com/jellyfin/jellyfin-plugin-fanart/blob/8bb2c0ddd4e5caf9a3ac5c1c894e8158040300f0/Jellyfin.Plugin.Fanart/Configuration/configPage.html#L16

Reply:
The current translation method is a complete hack, so we shouldn't encourage plugins to be using it right now in my opinion.